desktop-src/updateorchestrator/index.md

```diff
@@ -36,3 +36,46 @@ The initial release of UpdateOrchestrator API as limited access feature is only
 
 Our aim is to improve the functionality of this API and reduce impact from multiple automatic software updaters on Windows. We would appreciate your input through this [**brief survey**](https://aka.ms/UOAPISurvey) to help us understand how UpdateOrchestrator API can better serve your developer needs.
 
+## For OEMs wanting to Expedite Applications 
+
+>[!IMPORTANT]
+> Expediting Applications is only available for OEMs and only applicable in select regions. Expediting an application may have negative impacts to the out of box experience for new devices. This functionality is only available on select client builds.
+
+OEMs can register an update to be expedited by writing a new key in the existing staging folder for expedited apps: HKLM\Software\Microsoft\WindowsUpdate\Orchestrator\UScheduler_OOBE 
+
+Each OEM provider registration is in the form of a SubKey (with a unique name that will be used to identify this expedited app), and a set of registry values to indicate specific options for this update. The content of each key is in the following format: 
+
+Key name:  Unique name for this expedited app 
+
+Under Key name OEMs will need to create two additional keys:
+- updaterPriority: Number to indicate relative priority of this update 
+- expeditedPayload: JSON blob with options for this specific updater
+
+Example: 
+{   "PFN": "PFNName", 
+    "Endpoint": “SSL_URI”, 
+    "AllowedInOobe": false, 
+    "MaxRetryCount": 3, 
+    “TimeoutDurationInMinutes”: 15, 
+    “ExcludedRegions”: [“CN”]   } 
+
+Overview of used terms: 
+| State  | Type | Key name | Description | 
+| ------ | ---- | -------- | ----------- |
+| Required | String | PFN | The Package Family Name of the app (ex MicrosoftTeams_example) |
```

Comment:
For the sample, can we use an actual valid PFN?  For example, the Store PFN?  Microsoft.WindowsStore_8wekyb3d8bbwe